packages/wdio-appium-service/index.js

```diff
@@ -0,0 +1,2 @@
+const AppiumLauncher = require('./build/launcher').default
+module.exports = new AppiumLauncher()
```

Comment:
I thin we can remove this file, it doesn't seem to be used

Reply:
I've deleted the file :+1: